.github/workflows/build.yml

```diff
@@ -10,25 +11,21 @@ jobs:
       matrix:
         go: ['1.19', '1.20']
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #v3.5.2
 
     - name: Set up Go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 #v3.5.0
```

Comment:
Confirmed v3.5.0 is 6edd4406fa81c3da01a34fa6f6343087c207a568.

However, v4.0.1 it out.  Any reason not to just use the latest?